src/libostree/ostree-deployment.c

```diff
@@ -338,6 +412,7 @@ ostree_deployment_get_origin_relpath (OstreeDeployment *self)
  * ostree_deployment_unlocked_state_to_string:
  *
  * Since: 2016.4
+ * Returns: (not nullable): Description of state
```

Comment:
I don't *think* it matters to the parser, though for consistency: usually `Since:` is at the bottom of the docstring.